utils/machine/machine.go

```diff
@@ -157,6 +160,7 @@ func GetTopology(sysFs sysfs.SysFs, cpuinfo string) ([]info.Node, int, error) {
 				lastNode = -1
 			}
 			lastThread = thread
+			continue
```

Comment:
This whole loop could use some cleanup. I think it would be much more efficient and readable to switch on `strings.TrimSpace(line[:strings.Index(line, ":")-1])`

That's sort of a separate change, so up to you where / whether you want to address it.